gradle.properties

```diff
@@ -17,3 +17,6 @@ systemProp.jdk.tls.client.protocols=TLSv1.2
 # java homes resolved by environment variables
 org.gradle.java.installations.auto-detect=false
 org.gradle.java.installations.fromEnv=JAVA_TOOLCHAIN_HOME,JAVA_HOME,RUNTIME_JAVA_HOME,JAVA19_HOME,JAVA18_HOME,JAVA17_HOME,JAVA16_HOME,JAVA15_HOME,JAVA14_HOME,JAVA13_HOME,JAVA12_HOME,JAVA11_HOME,JAVA8_HOME
+
+# log some dependency verification info to console
+org.gradle.dependency.verification.console=verbose
```

Comment:
Do we want to keep this at verbose when we merge?

Reply:
I think we do. It's actually only `verbose` if a mismatch was detected and allows fixing this from the info you get from the command line without the need to look at the generated html report